internal/websocket/ws.go

```diff
@@ -187,7 +187,7 @@ func (ws *Websocket) HandleWrp(m wrp.Message) error {
 
 // AddMessageListener adds a message listener to the WS connection.
 // The listener will be called for every message received from the WS.
-func (ws *Websocket) AddMessageListener(listener event.MsgListener, cancel ...*event.CancelFunc) event.CancelFunc {
```

Comment:
I meant to remove `cancel ...*event.CancelFunc` when we integrate the wrphandlers